valhalla/jawn/src/managers/score/ScoreManager.ts

```diff
@@ -0,0 +1,59 @@
+import { dbExecute } from "../../lib/shared/db/dbExecute";
+import { err, Result } from "../../lib/shared/result";
+import { BaseManager } from "../BaseManager";
+
+export class ScoreManager extends BaseManager {
+  public async addScores(
+    requestId: string,
+    scores: Record<string, number>
+  ): Promise<Result<string, string>> {
+    try {
+      const organizationId = this.authParams.organizationId;
+      const scoreKeys = Object.keys(scores);
+      const scoreValues = Object.values(scores);
+
+      const { error: requestError } = await dbExecute(
+        `SELECT id FROM request WHERE id = $1 AND helicone_org_id = $2`,
+        [requestId, organizationId]
+      );
+
+      if (requestError) {
```

Comment:
We need to check if the request result is empty and error

Reply:
@chitalian , itt's still works fine and support older ways 
![image](https://github.com/Helicone/helicone/assets/38397684/7639aadd-ff63-4ee9-a9c9-d085ad9c0839)